sdk/python/v1beta1/kubeflow/katib/api_client.py

```diff
@@ -304,8 +320,12 @@ def __deserialize(self, data, klass):
             # convert str to class
             if klass in self.NATIVE_TYPES_MAPPING:
                 klass = self.NATIVE_TYPES_MAPPING[klass]
-            else:
+            elif klass in dir(kubeflow.katib.models):
                 klass = getattr(kubeflow.katib.models, klass)
+            elif klass in dir(kubernetes.client.models):
```

Comment:
I have to add this feature to support deserializing kuberentest's api resources such as V1ObjectMeta, ...

Reply:
Please can you try to run `make generate` to check if `api_client.py` will be properly generated with your change ?
We might need to do some modification in [`post_gen.py`](https://github.com/kubeflow/katib/blob/master/hack/gen-python-sdk/post_gen.py).

Reply:
Thanks @andreyvelich ,
I tried `make generate` and it erase all differences within this PR, as you might worried...